Update ap-stats.js (#1)

Heistergand opened the pull request:
* Update ap-stats.js
  - Take machina into account
  - Fix issue #879
  - Convert coffee into code
  - Add AP for reclaiming from machina

* Update total-conversion-build.js
  - Update and supplement ingress constants for access points
  - Change version to 0.42.0 
  - Celebrate the answer to the ultimate question of life, the universe, and everything

Heistergand committed:
Update ap-stats.js (#1)

* Update ap-stats.js

- take machina into account
- fix issue #879

xscreach reviewed (commented):
Reclaiming machina portal is not take into account 🤷

Heistergand committed:
Heistergand patch ap statistics plugin (#2)

* Update ap-stats.js

take machina into account, fix issue #879

* Update ap-stats.js

fix eslint

* Update ap-stats.js

convert coffe into code

* Update ap-stats.js

Heistergand committed:
Heistergand patch ap statistics plugin (#3)

* Update ap-stats.js

take machina into account, fix issue #879
convert coffe into code
Add AP for reclaiming from machina

* Update total-conversion-build.js

Update and supplement ingress constants for access points
changed version to 0.42.0

Heistergand committed:
Heistergand patch ap statistics plugin (#4)

* Update ap-stats.js
Take machina into account
fix issue #879
convert coffe into code
Add AP for reclaiming from machina

* Update total-conversion-build.js
Update and supplement ingress constants for access points
changed version to 0.42.0 -- celebrating the Answer to the Ultimate Question of Life, the Universe, and Everything

Heistergand committed:
Update total-conversion-build.js

Add CREATE_CONTROL_FIELD constant

xscreach reviewed (approved):
Nice
